### PR TITLE
Replace large configs

### PR DESCRIPTION
**Pull request by KieranHons**

### CODE REVIEW

File: config/test.json [hunk 1]
```diff
@@ -55,10 +55,12 @@
     }
   },
   "http": {
-    "enabled": false,
+    "enabled": true,
```
Comment (kbristow) [inline config/test.json]: This shouldnt be necessary for the test.json file I think.

File: templates/openshift/openshift-environment-setup.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<com.cloudbees.hudson.plugins.folder.Folder plugin="cloudbees-folder@6.0.4">
+    <description>Folder for {{projectName}} project [[managed by Subatomic]({{docsUrl}}/projects/{{projectName}})]</description>
```
Comment (kbristow) [inline templates/openshift/openshift-environment-setup.xml]: Is this actually a link?

File: src/openshift/OCClient.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export class OCClient {
     public static policy = OCPolicy;
 
     public static getInstance(): OCClient {
-        if (this.instance === null) {
+        if (this.instance === null || this.instance === undefined) {
```
Comment (kbristow) [inline src/openshift/OCClient.ts]: Try see if _.isEmpty(this.instance) works. You can find examples of this in other places in the repo.

File: src/openshift/OCCommon.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {StandardOption} from "./base/options/StandardOption";
 export class OCCommon {
 
     public static getInstance(): OCCommon {
-        if (this.instance === null) {
+        if (this.instance === null || this.instance === undefined) {
```
Comment (kbristow) [inline src/openshift/OCCommon.ts]: Same as above.

File: src/openshift/OCPolicy.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ import {StandardOption} from "./base/options/StandardOption";
 export class OCPolicy {
 
     public static getInstance(): OCPolicy {
-        if (this.instance === null) {
+        if (this.instance === null || this.instance === undefined) {
```
Comment (kbristow) [inline src/openshift/OCPolicy.ts]: Same as above.

File: templates/openshift/openshift-environment-setup.xml (hunk 2 above)
Comment (KieranHons) [reply templates/openshift/openshift-environment-setup.xml]: Wanted to chat about this, from the template each and every project would need to update the docs for this link to work. Is this something we actually want? Granted the visibility is great but don't know whether it'll be feasible or not

File: templates/openshift/openshift-environment-setup.xml [hunk 6]
```diff
@@ -1,5 +1,4 @@
 <com.cloudbees.hudson.plugins.folder.Folder plugin="cloudbees-folder@6.0.4">
-    <description>Folder for {{projectName}} project [[managed by Subatomic]({{docsUrl}}/projects/{{projectName}})]</description>
```
Comment (kbristow) [inline templates/openshift/openshift-environment-setup.xml]: Maybe we still want the description? Just without the link?
"Folder for {{projectName}} project (managed by Subatomic)"

Comment (KieranHons) [reply templates/openshift/openshift-environment-setup.xml]: Fair shout :)

**Pull request merged**